prep-css-layout-classes/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Layout Classes</title>
+  <link rel="stylesheet" href="reset.css">
+  <style>
+    /* stylelint-disable block-no-empty */
+
+    body {
+      font-family: sans-serif;
+    }
+
+    h1 {
+      text-align: center;
+    }
+
+    .container {
+      max-width: 992px;
+      margin: auto;
+    }
+
+    .row {
+      width: 100%;
+      display: flex;
+      justify-content: center;
+    }
+
+    .column {
+      flex-basis: 33%
+      float: left; padding-left: 8px;
+      padding-right: 8px;
+    }
+
+    h3.column {
+      text-align: center;
+      color: grey;
+    }
+
+    img.column {
+      width: 100%;
+    }
+
+  </style>
+</head>
+<body>
+
+  <div class="container">
+    <h1>Indiana Jonesssssss</h1>
+    <div class="row">
+      <div class="column">
+        <img class="column" src="file:///Users/anna/Downloads/prep-css-layout-classes/images/raiders.jpg">
```

Comment:
Try to avoid using URLs that include your file system details, like username, Downloads, etc.